source/gloperate-qtapplication/source/Viewer.cpp

```diff
@@ -23,8 +23,10 @@
 #include <gloperate-qt/QtMouseEventProvider.h>
 #include <gloperate-qt/QtWheelEventProvider.h>
 
-#include <gloperate-assimp/AssimpMeshLoader.h>
-#include <gloperate-assimp/AssimpSceneLoader.h>
+#ifdef GLOPERATE_ASSIMP_FOUND
```

Comment:
There is an `#ifdef ` missing surrounding `using namespace gloperate_assimp` because it will cause errors otherwise